web/src/helpers/api.ts

```diff
@@ -44,16 +44,20 @@ export function getUserList() {
   return axios.get<ResponseObject<User[]>>("/api/user");
 }
 
+export function getUserNameById(id: number) {
```

Comment:
We can just write a `getUserById`, maybe some fields of user might be useful later not only his name.